.pre-commit-config.yaml

```diff
@@ -8,8 +8,8 @@ repos:
   hooks:
   - id: black
 
-- repo: https://github.com/asottile/blacken-docs
-  rev: v1.12.1
+- repo: https://github.com/adamchainz/blacken-docs
+  rev: 1.16.0
```

Comment:
The updated versions in this file are from syncing up with what pymapdl is using as that is one of the more actively maintained repos.